tests/test_pipelines.py

```diff
@@ -476,7 +482,7 @@ def test_stable_diffusion_inpaint(self):
         image_slice = image[0, -3:, -3:, -1]
 
         assert image.shape == (1, 32, 32, 3)
-        expected_slice = np.array([0.4731, 0.5346, 0.4531, 0.6251, 0.5446, 0.4057, 0.5527, 0.5896, 0.5153])
+        expected_slice = np.array([0.3809, 0.5444, 0.4395, 0.413, 0.576, 0.4224, 0.4912, 0.5635, 0.422])
```

Comment:
Why the changes here? Does it give a different result on GPU? Should we force the test to be just CPU at the moment maybe?

Reply:
yes so it is different on CPU -vs- GPU.... I can make it CPU only?

Reply:
Yes we should indeed do this - think @anton-l  actually merged exactly this change yesterday: https://github.com/huggingface/diffusers/commit/472425098028edc6b5689986127a91ed5addd408

Sorry for the back and forth here

Reply:
thanks gonna add his changes here then